pkg/controllers/status/cluster_status_controller.go

```diff
@@ -364,14 +365,15 @@ func getAPIEnablements(clusterClient *util.ClusterClient) ([]clusterv1alpha1.API
 			if strings.Contains(resource.Name, "/") {
 				continue
 			}
-
 			apiResource := clusterv1alpha1.APIResource{
 				Name: resource.Name,
 				Kind: resource.Kind,
 			}
-
 			apiResources = append(apiResources, apiResource)
 		}
+		sort.SliceStable(apiResources, func(i, j int) bool {
+			return apiResources[i].Name < apiResources[j].Name
+		})
 		apiEnablements = append(apiEnablements, clusterv1alpha1.APIEnablement{GroupVersion: list.GroupVersion, Resources: apiResources})
```

Comment:
It makes sense.
Does the `apiEnablements` need to be sorted as well?

Reply:
+1, I'm not sure whether the sequence of `apiEnablements` is stable as well.

Reply:
ok